### PR TITLE
Make plan statistics thrift compatible

### PR DESCRIPTION
**Pull request by pranjalssh**
Tag HBO stats store specific structs with @ Thrift. These can then be serialized and stored in a database

```
== NO RELEASE NOTE ==
```

### CODE REVIEW

File: presto-spi/src/main/java/com/facebook/presto/spi/statistics/PlanStatistics.java [hunk 1]
```diff
@@ -29,6 +36,7 @@ public static PlanStatistics empty()
         return EMPTY;
     }
 
+    @ThriftConstructor
```
Comment (arunthirupathi) [inline presto-spi/src/main/java/com/facebook/presto/spi/statistics/PlanStatistics.java:39]: nit: requireNonNull(outputSize, "rowCount can not be null");

the error message should outputSize is nulll

**Review by arunthirupathi — APPROVED** [review]

**Review by highker — APPROVED** [review]
lgtm % arun's comments

**Comment by pranjalssh** [comment]
Updated

**Review by ajaygeorge — APPROVED** [review]
LGTM

**Pull request merged**